Node.js/README.md

```diff
@@ -6,49 +6,67 @@ Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine.
 
 ### Using Homebrew
 
-    brew install node
+```sh
+brew install node
+```
 
 ### Using Node Version Manager (nvm)
 
 Download and install [nvm](https://github.com/creationix/nvm) by running:
 
-    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash
+```sh
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash
+```
 
 Then download Node and select your version by running:
 
-    source ~/.bashrc        # source your bashrc/zshrc to add nvm to PATH
-    command -v nvm          # check the nvm use message
-    nvm install node        # install most recent Node stable version
-    nvm ls                  # list installed Node version
-    nvm use node            # use stable as current version
-    nvm ls-remote           # list all the Node versions you can install
-    nvm alias default node  # set the installed stable version as the default Node
+```console
+$ source ~/.bashrc        # source your bashrc/zshrc to add nvm to PATH
+$ command -v nvm          # check the nvm use message
+$ nvm install node        # install most recent Node stable version
+$ nvm ls                  # list installed Node version
+$ nvm use node            # use stable as current version
+$ nvm ls-remote           # list all the Node versions you can install
+$ nvm alias default node  # set the installed stable version as the default Node
+```
```

Comment:
```suggestion
```sh
source ~/.bashrc        # source your bashrc/zshrc to add nvm to PATH
command -v nvm          # check the nvm use message
nvm install node        # install most recent Node stable version
nvm ls                  # list installed Node version
nvm use node            # use stable as current version
nvm ls-remote           # list all the Node versions you can install
nvm alias default node  # set the installed stable version as the default Node
```
```